docs/js/index.js

```diff
@@ -9,6 +9,7 @@ function parallax() {
   } else {
     $.css(".logo", {
       transform: "translateY( -" + $.scrollTop() / 2.5 + "px)",
+      opacity: 1 - $.scrollTop() / 800,
```

Comment:
`800` is too small I think, 🤔, tell me if you would be ok with something like `5000`

Reply:
If it is `5000`, I will not notice the fade out animation(the opacity from 1 to about 0.85)
![shinyfaintibisbill-size_restricted](https://user-images.githubusercontent.com/16360246/32177481-6a61737c-bdc5-11e7-88e4-49bc0b092c9e.gif)

But if you really like `5000`, I can change it.

Reply:
Sorry, 😐 I made a mistake, when I tested the value, the dev console was hiding 70% of the height, 800 is ok 👌